Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorSelectionUtil.cpp

```diff
@@ -95,11 +95,12 @@ namespace AzToolsFramework
             entityId,
             [mouseInteraction, &entityPicked, &closestDistance, viewportId](EditorComponentSelectionRequests* handler) -> bool
             {
-                if (handler->SupportsEditorRayIntersect())
+                const auto viewportInfo = AzFramework::ViewportInfo{ viewportId };
```

Comment:
`viewportInfo` is not used outside the `if`, seems like it could be moved inside to reduce the scope where `viewportInfo` is seen.

Reply:
`viewportInfo` is passed in as an argument to `SupportsEditorRayIntersectViewport` which is why it's outside right now

Reply:
I missed that one, sorry.